src/rez/rezconfig.py

```diff
@@ -619,6 +619,29 @@
 suite_alias_prefix_char = "+"
 
 
+###############################################################################
+# Utils
```

Comment:
I would just add this to the `Misc` section, it's not that clear what "utils" really means in the context of rez settings.